lib/src/snapd_client.g.dart

```diff
@@ -0,0 +1,466 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
```

Comment:
Does this have to be checked in?

Reply:
It's recommended that published library packages commit generated files. The generated files are not only needed when developing it locally but also when publishing to pub.dev. Furthermore, leaving them out would prevent using the package as a Git dependency which is useful for experimenting with unpublished development versions.

I have an easy-to-remember alias for it but the full syntax for running build_runner is a bit clumsy so I added a Makefile to make it straightforward to regenerate the files.